packages/web/src/ui/styles/breakpoints.ts

```diff
@@ -1,3 +1,25 @@
+import { useState, useEffect } from 'react'
+
+const useMobile = () => {
```

Comment:
As mentioned above; this hook is unnecessary. Please remove and replace with Chakra Array style syntax: https://chakra-ui.com/docs/styled-system/features/responsive-styles#the-array-syntax